internal/git/git.go

```diff
@@ -300,3 +300,15 @@ func GetLocalRemotesFromFile() (string, error) {
 
 	return string(remotes), nil
 }
+
+func GetUnifiedDiff(BaseSHA string, HeadSHA string, oldPath string, newPath string) (string, error) {
+	// I hate magic numbers as much as the next person but I cannot
+	// figure out a better way to get a unified diff for an entire file.
```

Comment:
Hey, what? Lol.. I wasn't aware of that either.. but it seems this might be the "best" (maybe the only, or at least the most obvious) option indeed.